src/tabs.ts

```diff
@@ -0,0 +1,258 @@
+import {LitElement, html} from 'lit';
+import {customElement, state, property} from 'lit/decorators.js';
+import {animate_defs, animationCSS, ResponsiveVal,
+        getResponsiveValues, parseAttrVal} from "./utils";
+import {styleMap} from 'lit/directives/style-map.js';
+
+@customElement('aalam-tabs')
+export class AalamTabs extends LitElement {
+    DEFAULT_VALUES:{ [key:string]: any } = {
+        animationDur: 100,
+        activecls: 'tab-active',
+        fashion: 'xs:row',
+        colsize: 'title:30%;body:70%',
+        internal_fashion: 'row',
+        fashion_style: {xs:'row'},
+        column_size:{title:'30%', body:'70%'}
+    }
```

Comment:
Better to assign the default values to the variables itself. Even if you want to retain this format, better to remove the private defaults within it.